server/controllers/cellMove/cellMoveViewResidentialLocation.ts

```diff
@@ -63,7 +65,21 @@ export default ({ locationService, prisonerCellAllocationService }: Params) =>
 
     const prisoners = await prisonerCellAllocationService.getInmates(systemClientToken, locationDesc, null, true)
 
```

Comment:
call to prisonerDetailsService.getPrisoners is what provides CAT data. 
This is then merged in with the other prisoner details obtained by prisonerCellAllocationService.getInmates